### DISCUSSION TITLE
Umami Cloud CSP error

### DISCUSSION THREAD

**Discussion by shryme** [umami-software/umami#2443] [Q&A] [answered]
Hi, when visiting my Umami Cloud: https://analytics.us.umami.is/websites/1234
I currently see many problems with the CSP not correctly configured,
<img width="539" alt="image" src="https://github.com/umami-software/umami/assets/6409783/79b05ad9-b93b-4773-88ab-f9bbbc7aa59d">
The CSP tries to load cloud.umami, wich is not configured inside de CSP (api.umami.is and self, wich is analytics.us.umami.is), wich makes the website unable to load data. Are there others with the same problem? What can we do so that umami can load our data?

Thanks!

**Comment by mikecao** [comment, marked as answer]
This is only due to Next.js prefetching data for the settings link. It should not affect the application behavior. Did you verify that you are sending stats from your website?

**Reply by shryme** [reply]
Finally my problem was because I was injecting the script from clouds.umami instead of us.umami, I don't know why it made me copy a wrong url but by taking it from my setting everythings seems to work fine even with theses errors. thanks you!
